src/audio/eq_iir/eq_iir.c

```diff
@@ -31,30 +31,31 @@
 #include <stdint.h>
 
 /* tracing */
-#define trace_eq(__e, ...) trace_event(TRACE_CLASS_EQ_IIR, __e, ##__VA_ARGS__)
-#define trace_eq_with_ids(comp_ptr, __e, ...)			\
+#define trace(__e, ...) trace_event(TRACE_CLASS_EQ_IIR, __e, ##__VA_ARGS__)
+#define trace_wi(comp_ptr, __e, ...)			\
 	trace_event_comp(TRACE_CLASS_EQ_IIR, comp_ptr,		\
 			 __e, ##__VA_ARGS__)
 
-#define tracev_eq(__e, ...) tracev_event(TRACE_CLASS_EQ_IIR, __e, ##__VA_ARGS__)
-#define tracev_eq_with_ids(comp_ptr, __e, ...)			\
+#define tracev(__e, ...) tracev_event(TRACE_CLASS_EQ_IIR, __e, ##__VA_ARGS__)
+#define tracev_wi(comp_ptr, __e, ...)			\
 	tracev_event_comp(TRACE_CLASS_EQ_IIR, comp_ptr,		\
 			  __e, ##__VA_ARGS__)
 
-#define trace_eq_error(__e, ...) \
+#define trace_er(__e, ...) \
```

Comment:
"er"? Too vague, is this ONLY to prevent lines too long? Then not OK in my view.

Reply:
Would tr_error() be better? Also when tracing an error could we omit string "error:" from them since we already have trace class for errors? All of these tricks help a lot to keep the lines <= 80 chars.

Reply:
I'd say leave it as "trace_eq_error". You can omit the "error: " prefix if you wish. And then format strings are not subject to the <=80 chars restriction anyway.

Reply:
OK, I'll revert and the long lines will be back. I hope the checkpatch.pl script can be in future updated to contain these rules relaxations. It would be nice to see again the column of all green check marks from CI.

Reply:
Yeah, @jajanusz attempted to update certain things already (please rebase to latest master, I'm not sure you are using that update) so some "line too long" warnings shouldn't exist.